### ISSUE TITLE
TX802 performances

### ISSUE THREAD

**Issue by probonopd** [BobanSpasic/MDX_PerfConv#7]
Would it be possible to also convert TX802 performances? Along these lines:

```
        wget -q "http://hsjp.eu/downloads/Dexed/Dexed_cart_1.0.zip"
        unzip -q Dexed_cart_1.0.zip
        bin/Linux/MDX_PerfConv --info -a "./Dexed_cart_1.0/Original Yamaha/TX802/TX802BankA1.SYX" -b "./Dexed_cart_1.0/Original Yamaha/TX802/TX802BankB1.SYX" -p "./Dexed_cart_1.0/Original Yamaha/TX802/TX802_Factory_Performances.syx"
        bin/Linux/MDX_PerfConv -c -n 65 -a "./Dexed_cart_1.0/Original Yamaha/TX802/TX802BankA1.SYX" -b "./Dexed_cart_1.0/Original Yamaha/TX802/TX802BankB1.SYX" -p "./Dexed_cart_1.0/Original Yamaha/TX802/TX802_Factory_Performances.syx"
        # wget "https://github.com/asb2m10/dexed/files/5360597/tx802bankb-fixed.zip"
        # unzip tx802bankb-fixed.zip
        # ...
```

**Comment by BobanSpasic** [comment]
TX802 Performance file SysEx format is ... an abomination.
Even the guys at SysQuest (MidiQuest) have the same meaning...

It is a biiig ASCII file. It needs to be converted to a binary (undocumented) and see if it is anything alike to the known formats.
Theoretically, it should contain VMEM, AMEM and its own PMEM.

**Comment by probonopd** [comment]
Uh. Oh. Looks like that format is still a "frontier" for the open source world. Not super urgent, but maybe one day...

**Comment by BobanSpasic** [comment]
Not commited yet

```
c:\Users\Bobby\Documents\Lazarus\MiniDexed_Projects\MDX_PerfConv\bin\Win_x64>MDX_PerfConv.exe -i -a 802PRG1.SYX
Unknown Yamaha DX dump type: 25
Calculated data size: 9856
DX7II Voice Bank Supplement - AMEM at position 13
Calculated data size: 1120
Real data size: 1120
Checksum match
DX7/DX9 Voice Bank - VMEM at position 1141
Calculated data size: 4096
Real data size: 4096
Checksum match
```
```
c:\Users\Bobby\Documents\Lazarus\MiniDexed_Projects\MDX_PerfConv\bin\Win_x64>MDX_PerfConv.exe -i -a TX802_Factory_Performances.syx
TX802 PMEM at position 6
Calculated data size: 168
Real data size: 11581
Data size mismatch
```

**Comment by BobanSpasic** [comment]
I am almost finished with it, but I can't find all the factory patches that are used in the performance file that I've found.
TX802_Factory_Performances.syx from the internet has references to 4 voice bank files. I have the first two (= Bank A, voices 1-64).
The problem is the Bank B, because all the downloads I've found are in VMEM format (4104 bytes), and not VMEM+AMEM (5239 bytes).
Can anyone help?
